packages/vscode-extension/test/unit/migration/data/js/others/replace-getContext-commentLine.input.js

```diff
@@ -1,10 +0,0 @@
-import * as microsoftTeams from "@microsoft/teams-js";
```

Comment:
Update CODEOWNERS since the folder path is changed.

Reply:
Updated